pkg/parser/parser_test.go

```diff
@@ -538,6 +538,17 @@ func TestAdminStmt(t *testing.T) {
 		{"admin set bdr role secondary", true, "ADMIN SET BDR ROLE SECONDARY"},
 		{"admin unset bdr role", true, "ADMIN UNSET BDR ROLE"},
 		{"admin show bdr role", true, "ADMIN SHOW BDR ROLE"},
+		// for alter job
+		{"admin alter ddl jobs 1 thread = 2", true, "ADMIN ALTER DDL JOBS 1 thread = 2"},
+		{"admin alter ddl jobs 1 thread = ", false, ""},
+		{"admin alter ddl jobs 1 thread", false, ""},
+		{"admin alter ddl jobs 1 batch_size = 3", true, "ADMIN ALTER DDL JOBS 1 batch_size = 3"},
+		{"admin alter ddl jobs 1 batch_size = ", false, ""},
+		{"admin alter ddl jobs 1 batch_size", false, ""},
+		{"admin alter ddl jobs 1 max_write_speed = 4", true, "ADMIN ALTER DDL JOBS 1 max_write_speed = 4"},
```

Comment:
what is the 4 means max_write_speed?

Reply:
The default unit is `byte`, here supports the style of omitting units.